apps/api/src/controllers/accounts.controller.ts

```diff
@@ -0,0 +1,63 @@
+import {
+  Controller,
+  Get,
+  Post,
+  HttpCode,
+  HttpStatus,
+  Logger,
+  Param,
+  HttpException,
+} from '@nestjs/common';
+import { ApiOkResponse, ApiOperation, ApiTags } from '@nestjs/swagger';
+import { AccountsService } from '../services/accounts.service';
+import { AccountResponse } from '../../../../libs/common/src/dtos/accounts.dto';
+
+@Controller('accounts')
+@ApiTags('account-service')
```

Comment:
Naming suggestion: How do we feel about `msa-service` for all msa-related api's?

Reply:
I changed them to the be the base enpoint name in the next story so this is now "accounts"